mne/viz/ica.py

```diff
@@ -1001,12 +1001,12 @@ def _plot_ica_overlay_evoked(evoked, evoked_cln, title, show):
     fig.suptitle(title)
     axes = axes.flatten() if isinstance(axes, np.ndarray) else axes
 
-    evoked.plot(axes=axes, show=False, time_unit='s')
+    evoked.plot(axes=axes, show=False, time_unit='s', spatial_colors=False)
```

Comment:
why are these necessary?

Reply:
I believe the reason is that in the overlay plots, the only colors we want are for differentiating between before and after data cleaning; spatial colors would make it more difficult to see what has changed

Reply:
Also, since the default was 'False' before, I thought not adding 'spatial_colors'=False would change the function's (_plot_ica_overlay_evoked_plot_ica_overlay_evoked) behavior.